index.js

```diff
@@ -1,64 +1,78 @@
-let idCounter = localStorage.getItem('counter')
-  ? Number(localStorage.getItem('counter'))
-  : localStorage.setItem('counter', 0);
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Please let's kindly take note that it's not a good practice to disable linters, one of the importance of linters is to identify certain programming errors earlier on that you may not have noticed while writing code. Perhaps if you place one of the two classes in a separate file, the linter issue will be resolved. ℹ️

Reply:
@OyePriscilla , 

Thank you for the comment.

There is a tip in the requirement to add this linter disabled please see the screenshot below.

![image](https://user-images.githubusercontent.com/35835135/226574290-baf92b5a-12aa-42be-ad29-91a5ae172a41.png)